src/index.html

```diff
@@ -11,7 +11,160 @@
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+            name="old"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="birth"
+            placeholder="mm/dd/yyyy"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement <input type="checkbox" name="term">
+        </label>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            name="email"
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="24"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="loveCat"
+              value="true"
+            > Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="loveCat"
+              value="false"
+            > No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            step="1"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+            name="cars"
+            size="4"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+            <option></option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rate"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>Additional info:</legend>
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="comments"
+            cols="20"
+            rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="comments"
            cols="20"
            rows="2"
        ></textarea>
```
![image](https://user-images.githubusercontent.com/96409563/221144311-a2f8680e-9bc8-4d18-b8b0-314a6958e01d.png)
![image](https://user-images.githubusercontent.com/96409563/221144397-f8745d15-57f5-4016-a914-e15453e21bbf.png)